script/convert_mesh.py

```diff
@@ -39,7 +39,7 @@
     'tri-tetra' : 'convert elements: quad->tri, hexa->tetra',
     '2d' : 'force a 2D mesh by removing the z coordinates - assumes a 3D mesh'
     ' in the xy plane',
-    '3d' : 'force a 3D mesh by removing 1D and 2D cells',
+    'cell-dim' : 'write only cells of a given dimension(s)',
```

Comment:
Maybe more explicitly:
"write only cells of a given dimension - use a comma-separated list for several values."

Reply:
updated